CONTRIBUTING.md

```diff
@@ -1,39 +1,65 @@
+even if you are a _developer reviewing your contribution_. **Sorry if we can't meet your expectations, we do our best**.
 
 - **To add, remove, or change things on the list:** Submit a pull request
 
-To set this list apart from and complement the excellent [Go wiki Projects page](https://golang.org/wiki/Projects), awesome-go is a specially curated list for high-quality, actively maintained Go packages and resources.
+To set this list apart from and complement the excellent [Go wiki Projects page](https://golang.org/wiki/Projects), 
+and other lists, awesome-go is a specially curated list for high-quality, actively maintained Go packages and resources.
 
-- List items should be sorted *alphabetically*;
-- Each item should be limited to one link;
-- The link should be the name of the package or project;
-- Descriptions should be clear, concise, and non-promotional;
-- Descriptions should follow the link, on the same line and end with a punctuation mark;
 - At least 3 items are needed to create a new category;
 - The package or project had to be maintained under **open source license** ( *we make a brief review of the code before the link enters the list* ), [see list of allowed licenses](https://opensource.org/licenses/alphabetical);
-- Remember to put a period `.` at end of the project description.
 
 Please contribute links to packages/projects you have used or are familiar with. This will help ensure high-quality entries.
 
 If you removed our PR template you can find it [here](https://github.com/avelino/awesome-go/blob/main/.github/PULL_REQUEST_TEMPLATE.md).
 
 
-## Quality standards
+## General quality standards
 
-To be on the list, project repositories should adhere to these quality standards (https://goreportcard.com/report/github.com/ **github_user** / **github_repo**):
+To be on the list, project repositories should adhere to these quality standards 
+(https://goreportcard.com/report/github.com/ **github_user** / **github_repo**):
 
-- Code functions as documented and expected
-- Generally useful to the wider community of Go programmers
+- Code functions as documented and expected;
+- Generally useful to the wider community of Go programmer
 - Actively maintained
   - Regular, recent commits
-  - Or, for finished projects, issues and pull requests are responded to
+  - Or, for finished projects, issues and pull requests are responded to generally within 2 weeks
 - Stable or progressing toward stable
 - Thoroughly documented (README, pkg.go.dev doc comments, etc.) in english language, so everyone is able to understand the project's intention and how it works
 - Tests, where practical. If the library/program is testable, then coverage should be >= 80% for non-data-related packages and >=90% for data related packages. **Notice**: the tests will be reviewed too. We will check your coverage manually if your package's coverage is just a benchmark results.
+- The project has a Go Report Card value of A.
```

Comment:
Then A+? I just put that in as a guess because I didn't see anywhere what the expectation was. If A+ is not good enough still, what is the expectation?

Reply:
it is not a matter of being A+ (or any other grade), if all items have OK and golint bad, even then the project can get an A+ grade

so I don't see it as a logical and clear rule

Reply:
OK. What should the rule be? What should maintainers be looking for on the report?